docker-compose.yml

```diff
@@ -19,13 +19,13 @@ services:
       - "8092:8091"
       - "${DEBUG_PORT}:5005"
     depends_on:
-       - ldap  
+       - ldap
     networks:
       - confluence-net
 
   postgres:
     image: postgres:${POSTGRESQL_VERSION}
-    container_name: postgres
+    container_name: postgres_${CONFLUENCE_VERSION}
```

Comment:
I used ${CONFLUENCE_VERSION} for all containers name suffix, so all are vinculated to the specific confluence version.